var/spack/repos/builtin/packages/vtk/package.py

```diff
@@ -322,7 +323,7 @@ def cmake_args(self):
             cmake_args.extend(
                 [
                     "-DVTK_USE_SYSTEM_LIBPROJ4:BOOL=OFF",
-                    "-DNETCDF_CXX_ROOT={0}".format(spec["netcdf-cxx"].prefix),
+                    "-DNETCDF_CXX_ROOT={0}".format(spec["netcdf-cxx4"].prefix),
```

Comment:
Wouldn't this fail for 6.1.0 (trying to access `netcdf-cxx4` while depending on `netcdf-cxx`)?

Reply:
Yeah, good catch, I think I screwed up a copy-paste up above.